src/MensaWidget/MWDataGettingWrapper.class.st

```diff
@@ -51,3 +67,83 @@ MWDataGettingWrapper class >> getTodaysMeals [
 	today := Date today yyyymmdd.
 	^ MWDataGettingWrapper getMealsOnDate: today
 ]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Oscar 7/1/2024 16:32'
+}
+MWDataGettingWrapper class >> initialLatitude [
+
+	^ InitialLatitude ifNil: [52.393822]
```

Comment:
I am personally a fan of filling in the value if its nil like done in the setup screen we need to come together and discuss this to unify

Reply:
^ InitialLatitude ifNil: [InitialLatitude := '52.393822'] you can simplify them like this then you dont need ifTrue: ifFalse

Reply:
You cannot simplify them, if the user doesn't enter any value it is going to be an empty string and then we call the url with no value

Reply:
that's why we need to check for an empty string